src/model/user-settings/UserSettings.ts

```diff
@@ -821,13 +821,14 @@ export class UserSettings implements IUserSettings {
         if (button) {
           addEventListenerOptional(button, "click", (event: MouseEvent) => {
             const position = this.view.getCurrentPosition();
-            this.userProperties.getByRef(ReadiumCSS.SCROLL_REF).value = index;
+            this.userProperties.getByRef(ReadiumCSS.SCROLL_REF).value =
+              index == 0 ? true : false;
             this.storeProperty(
               this.userProperties.getByRef(ReadiumCSS.SCROLL_REF)
             );
             this.applyProperties();
             this.updateViewButtons();
-            this.view.setMode(index === 0);
+            this.view.setMode(index == 0 ? true : false);
```

Comment:
In general I think we should use `===` instead of `==` since `==` does type coercion in sometimes unexpected ways. For example:
```
false == 0 // true
"NaN" == 0 // true
undefined == 0 // true

false === 0 // false
"NaN" === 0 // false
undefined === 0 // false
0 === 0 // true
```

Reply:
you're right, i already changed that in some other places.